solidity/dashboard/src/pages/RewardsPage.jsx

```diff
@@ -1,13 +1,8 @@
 import React from "react"
 import { Rewards } from "../components/Rewards"
-import PageWrapper from "../components/PageWrapper"
 
 const RewardsPage = () => {
-  return (
-    <PageWrapper title="Random Beacon Rewards">
-      <Rewards />
-    </PageWrapper>
-  )
+  return <Rewards />
```

Comment:
Why have a `RewardsPage` intermediary in the first place if all it does is pass through the component, out of curiosity?

Reply:
Writing this down while it's on my mind: keeping the [previous note](https://github.com/keep-network/keep-core/pull/2101#discussion_r500639319) around link data and the routing table in mind, I'm wondering if we reconsider some of our component vs page distinctions a little bit. In particular, my thinking here would be:
- Pages are defined as React components that bundle page structure + routing information.
- Anything that requires routing details (e.g., `ResourcesLink`) should either be used directly at the `Page` level, or it should be passed to the component that needs it (as a prop).
- A Page component is a function, but it has a special property, `route`, that indicates routing information (title, path, and `pages`, which specifies child `Page` components).
- The routing table is a list of top-level `Page` components, which in turn may define recursive children.

Quick sketch of this:
```
const pages = [OverviewPage, TokensPage, …, ApplicationsPage, …]

...

const ApplicationsPage = (...) {
  …
}
ApplicationsPage.route = {
  title: 'Applications',
  path: '/applications/',
  pages: [KeepRandomBeaconApplicationPage, TBTCApplicationPage]
}

…

const KeepRandomBeaconApplicationPage = (…) {
  …
}
KeepRandomBeaconApplicationPage.route = {
  title: 'Keep Random Beacon,
  path: '/random-beacon
}
```

Then we could use this to build up the routing table. There's a similar concept defined on the react-router site @ https://reactrouter.com/web/example/route-config . Thoughts?